svo (actions dictionary).xml

```diff
@@ -13767,7 +13967,95 @@ if svo.haveskillset('ignition') then
       }
     }
     end
+    
+if svo.haveskillset('memorium') then
+  svo.dict.traceshield = {
+  gamename = 'shieldlogograph',
+    physical = {
+      balanceful_act = true,
+      def = true,
+      
+        isadvisable = function ()
+        return (defc.bloodiedknife and not defc.traceshield and ((sys.deffing and defdefup[defs.mode].traceshield) or (conf.keepup and defkeepup[defs.mode].traceshield)) and not codepaste.balanceful_defs_codepaste() and not affs.paralysis and not affs.prone) or false
+      end,
+      
+        oncompleted = function ()
+          defences.got('traceshield')
+        end,
+        
+      actions = {"trace shield me"},
+      onstart = function ()
+        send('trace shield me', conf.commandecho)
+      end
+      }
+    }
+    
+    svo.dict.traceleech = {
+    gamename = 'leechlogograph',
+    physical = {
+      balanceful_act = true,
+      def = true,
+      
+        isadvisable = function ()
+        return (defc.bloodiedknife and not defc.traceleech and ((sys.deffing and defdefup[defs.mode].traceleech) or (conf.keepup and defkeepup[defs.mode].traceleech)) and not codepaste.balanceful_defs_codepaste() and not affs.paralysis and not affs.prone) or false
+      end,
+      
+        oncompleted = function ()
+          defences.got('traceleech')
+        end,
+        
+      actions = {"trace leech me"},
+      onstart = function ()
+        send('trace leech me', conf.commandecho)
+      end
+      }
+    }
+    end
```

Comment:
The end on line 14013 closes the if statement checking if you have the memorium skillset. Making it so bloodiedknife is permanently in the dict. If we move it below the bloodiedknife entry it will fix the issue without the need for the class check.